client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/style.scss

```diff
@@ -221,22 +221,10 @@ $design-button-primary-color: rgb(17, 122, 201);
 		}
 
 		.step-container__content {
-			height: calc(100vh - 245px);
```

Comment:
Now the `@automattic/design-preview` component has the following mode so we don't need to specify the height here.
* default: The position is relative, and the height grows automatically.
* fullscreen: The position is fixed, and the height is the same as the viewport height reducing the height of the navigation footer)

The fullscreen mode is used only when the user is going to select Styles/Colors/Fonts on mobile.